### DISCUSSION TITLE
Adoption

### DISCUSSION THREAD

**Discussion by xshesinnedx** [Luke100000/minecraft-comes-alive#821] [Ideas]
Would it ever be possible to bring back the adoption from the original minecraft comes alive? I would love to be able to adopt children through the cleric/priest again. I feel kinda bad when i find a village with an abandoned baby lol. And it really adds to the story when kind of roleplaying through your own little world. Just an idea/suggestion/question. Thank you for all the amazing work you do, this is my favorite mod, im so glad it was updated! You are my minecraft modding hero lol! Much love! <3

**Comment by Luke100000** [comment]
You can now adopt any orphan child by talking to them

**Reply by xshesinnedx** [reply]
Thank you so much!! <3
